src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentalCost = 40 * days;
+
+  if (days >= 3 && days < 7) {
+    return rentalCost - 20;
+  } else if (days >= 7) {
+    return rentalCost - 50;
+  }
```

Comment:
1. make it clearer.
2. move all magic numbers into a constants
```suggestion
  if ( && days >= 7) {
    return rentalCost - biggerDiscount;
  }
  
    if (days >= 3) {
    return rentalCost - smallerDiscount;
  }
```